examples/examples.json

```diff
@@ -113,6 +113,21 @@
                 }
             ]
         },
+        {
+            "title": "Sprites",
+            "samples": [
+                {
+                    "title": "Single Sprite",
+                    "desc": "Show a sprite in the place of points",
+                    "file": "sprites/sprite.html"
+                },
+                {
+                    "title": "Multiple Sprites",
+                    "desc": "Show different sprites based on a category",
```

Comment:
Show different sprites based on **category values**.

Reply:
:+1:

Reply:
`based on a category property`?

Reply:
Sure, that works too.